src/SFA.DAS.AssessorService.Web/Views/Standard/OptOutStandardVersion.cshtml

```diff
@@ -14,18 +14,19 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
 
+            @if (!ViewData.ModelState.IsValid)
+            {
+                <partial name="~/Views/Standard/_OptOutValidationErrorPartial.cshtml" model=@Model />
```

Comment:
partials are great if you are using them multiple times, but they make the code less readable - so avoid them if it's a one off

Reply:
Removed